### PR TITLE
readme

### PR DESCRIPTION
**Pull request by pierrebrunelle**
- Simplified first Answer to What problesm does Pixeltable solve? And moved remaining sentences to FAQ below
- Update Header 
- Added Getting Started Notebook, Removed API Reference Column, Added Label Studio Link + New Button
- Added Code Samples with link to relevant notebooks where snippets is extracted from
- Remove Example of Use Cases section

### CODE REVIEW

**Comment by aaron-siegel** [comment]
This change is [Reviewable](https://reviewable.io/reviews/pixeltable/pixeltable/225)

File: README.md [hunk 1]
```diff
@@ -5,47 +5,134 @@
...
-## What problems does Pixeltable solve?
+### What problems does Pixeltable solve?
 
-Today’s solutions for AI app development require extensive custom coding and infrastructure plumbing. Tracking lineage and versions between and across data transformations, models, and deployment is cumbersome. With Pixeltable you can store, transform, index, and iterate on your data within the same table interface, whether it's text, images, embeddings, or even video. Built-in lineage and versioning ensure transparency and reproducibility, while the development-to-production mirror streamlines deployment.
+Today’s solutions for AI app development require extensive custom coding and infrastructure plumbing. Tracking lineage and versions between and across data transformations, models, and deployment is cumbersome.
 
 ## 💾 Installation
 
 ```python
 %pip install pixeltable
```
Comment (aaron-siegel) [inline README.md]: Note this won't work from the commandline. I'd recommend we leave out the `%`; it's not really an "installation" to run this command in a notebook anyway. And most Jupyter users will know to put the `%` or `!` before the `pip`.

File: README.md [hunk 2]
```diff
@@ -5,47 +5,134 @@
...
+> [!NOTE]
+> Check out the [Pixeltable Basics](https://pixeltable.readme.io/docs/pixeltable-basics) tutorial for a tour of its most important features.
+
+## 💡 Getting Started
+Learn how to create tables, populate them with data, and enhance them with built-in or user-defined transformations and AI operations.
 
-To verify that it's working:
+| Topic | Notebook |
+|:--------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+| 10-minute tour of Pixeltable    | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/tutorials/pixeltable-basics.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
```
Comment (aaron-siegel) [inline README.md]: Capitalize the title to be consistent with the others ("10-Minute Tour of Pixeltable")

File: README.md [hunk 3]
```diff
@@ -5,47 +5,134 @@
...
+| Comparing Object Detection Models | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/tutorials/object-detection-in-videos.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| Experimenting with Chunking (RAG) | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/tutorials/rag-operations.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>
+| Working with External Files    | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/howto/working-with-external-files.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| Integrationg with Label Studio for Annotations    | <a target="_blank" href="https://pixeltable.readme.io/docs/label-studio"> <img src="https://img.shields.io/badge/Docs-Label Studio-blue" alt="Visit our documentation"/></a>
```
Comment (aaron-siegel) [inline README.md]: "Integrating" is misspelled here

File: README.md [hunk 4]
```diff
@@ -5,47 +5,134 @@
...
 
+## 🧱 Code Samples
+
+### All media data (videos, images, audio) resides in external files, and Pixeltable stores references to those
```
Comment (aaron-siegel) [inline README.md]: How about "Import media data into Pixeltable" instead? The fact that "Pixeltable stores references to those" is kinda distracting in the heading, and it's addressed in the code comment in the example in any case

File: README.md [hunk 5]
```diff
@@ -5,47 +5,134 @@
...
+paths = [
+    'data/videos/mp4/ffe/ffb/ffeffbef41bbc269810b2a1a888de.mp4',
+    'data/videos/mp4/ffe/feb/ffefebb41485539f964760e6115fbc44.mp4',
+    'data/videos/mp4/ffe/f73/ffef7384d698b5f70d411c696247169.mp4'
+]
+v.insert({'video': prefix + p} for p in paths)
 ```
-> [!NOTE]
-> Check out the [Pixeltable Basics](https://pixeltable.readme.io/docs/pixeltable-basics) tutorial for a tour of its most important features.
+Learn more on how to [access the original file paths and work with your data](https://pixeltable.readme.io/docs/working-with-external-files).
```
Comment (aaron-siegel) [inline README.md]: I don't understand what "access the original file paths and work with your data" means

File: README.md [hunk 6]
```diff
@@ -5,47 +5,134 @@
...
+| Experimenting with Chunking (RAG) | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/tutorials/rag-operations.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>
+| Working with External Files    | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/howto/working-with-external-files.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| Integrationg with Label Studio for Annotations    | <a target="_blank" href="https://pixeltable.readme.io/docs/label-studio"> <img src="https://img.shields.io/badge/Docs-Label Studio-blue" alt="Visit our documentation"/></a>
 
+## 🧱 Code Samples
+
+### All media data (videos, images, audio) resides in external files, and Pixeltable stores references to those
 ```python
```
Comment (aaron-siegel) [inline README.md:44]: Add `import pixeltable as pxt`

File: README.md [hunk 7]
```diff
@@ -5,47 +5,134 @@
...
+]
+v.insert({'video': prefix + p} for p in paths)
 ```
-> [!NOTE]
-> Check out the [Pixeltable Basics](https://pixeltable.readme.io/docs/pixeltable-basics) tutorial for a tour of its most important features.
+Learn more on how to [access the original file paths and work with your data](https://pixeltable.readme.io/docs/working-with-external-files).
 
-## 💡 Get Started
-Learn how to create tables, populate them with data, and enhance them with built-in or user-defined transformations and AI operations.
+### Create a computed column based on a function calling an object detection model
```
Comment (aaron-siegel) [inline README.md]: How about "Add an object detection model to a workflow" or some such?

File: README.md [hunk 8]
```diff
@@ -5,47 +5,134 @@
...
+```python
+# Pixeltable is downloading the model from Huggingface (if necessary), instantiating it, and caching it for later use
```
Comment (aaron-siegel) [inline README.md]: Not sure this comment is helpful here.

Also, add `from pixeltable.functions import huggingface`

File: README.md [hunk 9]
```diff
@@ -5,47 +5,134 @@
...
+t['detections'] = huggingface.detr_for_object_detection(t.input_image, model_id='facebook/detr-resnet-50')
+```
+Learn more on how to use computed colums through the example of [comparing object detection models](https://pixeltable.readme.io/docs/object-detection-in-videos).
```
Comment (aaron-siegel) [inline README.md]: "columns" is misspelled.

I suggest:
```
Learn more about computed columns and object detection: [Object Detection in Videos](...)
```

File: README.md [hunk 10]
```diff
@@ -5,47 +5,134 @@
...
+
+### Build User-Defined Functions to expand Pixeltable
```
Comment (aaron-siegel) [inline README.md]: Either capitalize "expand" (pls let's try to use consistent capitalization in our docs) or don't capitalize "User-Defined Functions"

I prefer "Extend Pixeltable's Capabilities" in place of "Expand Pixeltable"

File: README.md [hunk 11]
```diff
@@ -5,47 +5,134 @@
...
+```python
+# The longest_word Python function isn't a Pixeltable UDF (yet); Adding the decorator turns it into a UDF:
+@pxt.udf
+def longest_word(sentence: str, strip_punctuation: bool = False) -> str:
+    words = sentence.split()
+    if strip_punctuation:  # Remove non-alphanumeric characters from each word
+        words = [''.join(filter(str.isalnum, word)) for word in words]
+    i = np.argmax([len(word) for word in words])
+    return words[i]
```
Comment (aaron-siegel) [inline README.md]: Not sure this is the best example to use here; it's really a toy example. I suggest substituting the bounding box drawing UDF from the object detection in videos tutorial.

File: README.md [hunk 12]
```diff
@@ -5,47 +5,134 @@
...
+### Create a computed column based on a function calling an object detection model
+```python
+# Pixeltable is downloading the model from Huggingface (if necessary), instantiating it, and caching it for later use
+t['detections'] = huggingface.detr_for_object_detection(t.input_image, model_id='facebook/detr-resnet-50')
+```
+Learn more on how to use computed colums through the example of [comparing object detection models](https://pixeltable.readme.io/docs/object-detection-in-videos).
+
+### Build User-Defined Functions to expand Pixeltable
+```python
+# The longest_word Python function isn't a Pixeltable UDF (yet); Adding the decorator turns it into a UDF:
```
Comment (aaron-siegel) [inline README.md]: This comment makes no sense here.

File: README.md [hunk 13]
```diff
@@ -5,47 +5,134 @@
...
+@pxt.udf
+def longest_word(sentence: str, strip_punctuation: bool = False) -> str:
+    words = sentence.split()
+    if strip_punctuation:  # Remove non-alphanumeric characters from each word
+        words = [''.join(filter(str.isalnum, word)) for word in words]
+    i = np.argmax([len(word) for word in words])
+    return words[i]
+```
+Learn more on how to leverage [UDFs](https://pixeltable.readme.io/docs/user-defined-functions-udfs)
```
Comment (aaron-siegel) [inline README.md]: As above (and in all these examples) I prefer to use the exact title of the tutorial for the link, as in:

```
Learn more about user-defined functions: [UDFs in Pixeltable](...)
```

File: README.md [hunk 14]
```diff
@@ -5,47 +5,134 @@
...
+
+### Automate data operations with Views
```
Comment (aaron-siegel) [inline README.md]: Don't capitalize "Views"

File: README.md [hunk 15]
```diff
@@ -5,47 +5,134 @@
...
+    iterator=DocumentSplitter.create(
+        document=documents_t.document,
+        separators='token_limit', limit=300)
+)
+```
+Learn more on how to leverage Views to build your [RAG workflow](https://pixeltable.readme.io/docs/document-indexing-and-rag).
+
+### Evaluate models performance
+```python
+# The computation of the mAP metric can simply become query over the evaluation output, aggregated with the mean_ap() function.
```
Comment (aaron-siegel) [inline README.md]: "can simply become a query"

File: README.md [hunk 16]
```diff
@@ -5,47 +5,134 @@
...
+    'rag_demo.chunks',
+    documents_t,
+    iterator=DocumentSplitter.create(
+        document=documents_t.document,
+        separators='token_limit', limit=300)
+)
+```
+Learn more on how to leverage Views to build your [RAG workflow](https://pixeltable.readme.io/docs/document-indexing-and-rag).
+
+### Evaluate models performance
```
Comment (aaron-siegel) [inline README.md]: models -> model

File: README.md [hunk 17]
```diff
@@ -5,47 +5,134 @@
...
+chat_t['response'] = chat_t.output.choices[0].message.content
+
+# Start a conversation
+chat_t.insert([
+    {'input': 'How many species of felids have been classified?'},
+    {'input': 'Can you make me a coffee?'}
+])
+chat_t.select(chat_t.input, chat_t.response).head()
+```
+Learn more on how to leverage LLms such as [Together AI](https://pixeltable.readme.io/docs/together-ai) in Pixeltable.
```
Comment (aaron-siegel) [inline README.md]: LLms -> LLMs

Together AI isn't an LLM though. Let's be accurate ("inference services such as Together AI" or "LLMs hosted by services such as Together AI" or some such)

File: README.md (hunk 1 above)
Comment (pierrebrunelle) [reply README.md]: Sounds good.

File: README.md (hunk 6 above)
Comment (pierrebrunelle) [reply README.md:44]: I think it's distracting. This is to focus on the code itself. All these snippets are missing pip install and import statements. None of them are runnable in isolation.

File: README.md (hunk 7 above)
Comment (pierrebrunelle) [reply README.md]: Ok and maybe I can add another code snippet just about "computed column".

File: README.md (hunk 8 above)
Comment (pierrebrunelle) [reply README.md]: Same as above. I left pip install and import statement out of these code snippets as even with those and they are not runnable by themselves and the goal is to focus the attention on the logic and the simplicity/flexibility of the code.

Comment (aaron-siegel) [reply README.md]: Ok, but can we get rid of the comment? I don't think it adds anything (it's cargo-culted from the notebook but I think it's confusing out of context)

File: README.md (hunk 6 above)
Comment (aaron-siegel) [reply README.md:44]: I wasn't suggesting adding pip install everywhere, but I'd advocate for at least putting it is just this one code snippet. It clarifies the meaning of `pxt`. This one actually *will* work on its own!

File: README.md [hunk 18]
```diff
@@ -5,47 +5,132 @@
...
+| Comparing Object Detection Models | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/tutorials/object-detection-in-videos.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| Experimenting with Chunking (RAG) | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/tutorials/rag-operations.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>
+| Working with External Files    | <a target="_blank" href="https://colab.research.google.com/github/pixeltable/pixeltable/blob/master/docs/release/howto/working-with-external-files.ipynb"> <img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> </a>
+| Integrating with Label Studio for Annotations    | <a target="_blank" href="https://pixeltable.readme.io/docs/label-studio"> <img src="https://img.shields.io/badge/Docs-Label Studio-blue" alt="Visit our documentation"/></a>
 
-To verify that it's working:
+## 🧱 Code Samples
```
Comment (aaron-siegel) [inline README.md:42]: We should probably put a comment somewhere to let ppl know that these code samples aren't runnable on their own

**Review by aaron-siegel — APPROVED** [review]
Approved with a few more comments

File: README.md (hunk 6 above)
Comment (pierrebrunelle) [reply README.md:44]: Done.

File: README.md (hunk 18 above)
Comment (pierrebrunelle) [reply README.md:42]: I feel like it's expected for people browsing our repo based on their skill sets to understand that these snippets require definition and setup to be runnable. Let's revisit that later.

**Comment by pierrebrunelle** [comment]
I incorporated the latest comments. Thanks for the review. Looks great.

**Pull request merged**